pkg/crc/services/dns/dns_windows.go

```diff
@@ -7,13 +7,21 @@ import (
 	"time"
 
 	"github.com/code-ready/crc/pkg/crc/errors"
+	"github.com/code-ready/crc/pkg/crc/output"
 	"github.com/code-ready/crc/pkg/crc/services"
 
 	"github.com/code-ready/crc/pkg/os/windows/powershell"
 	"github.com/code-ready/crc/pkg/os/windows/win32"
 )
 
 func runPostStartForOS(serviceConfig services.ServicePostStartConfig, result *services.ServicePostStartResult) (services.ServicePostStartResult, error) {
+	// bailout for Virtualbox
+	if serviceConfig.Driver.DriverName() == "virtualbox" {
+		output.Out("Please follow instructions in the documentation about setting hostnames for Virtualbox.")
```

Comment:
I'm not sure this is going to be very visible?
So when vbox networking was not manually configured, the user will see this message, but crc start will still keep going as if everything went well, but things won't work? Or do we have a postfilght check after this one which will check that network is properly configured, and fail if not?

Reply:
We will not do any automated setups. They already find the information about using virtulabox mostly from the documentation. And in this case, what else can we do? Currently we fail... and this will create a cluster but errors. With this change at least the command will succeed. The user has to perform manual steps ... we also can not perform postflight checks, as the user is responsible and only gets the IP address to make these changes after the VM started and with `crc ip`.

Reply:
To be sure I understand correctly, the workflow after your patch will be:
- user runs `crc start`
- message is printed to tell the user to read the doc
- start command still keeps going but with misc failures because DNS is not setup correctly
- user run `crc ip` and setups DNS
- user runs `crc start` again

If yes, couldn't we do instead "if driver is virtualbox, try to resolve api.crc.testing, and check this resolves to the VM IP, if this fails, this is a hard failure, instruct to follow instructions, and print VM IP address"